docker/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+services:
+  chat-copilot-webapp:
+    image: chat-copilot-webapp-nginx
+    build: 
+      context: ..
+      dockerfile: docker/webapp/Dockerfile
+    ports:
+      - 3000:3000
+    env_file:
+      - ../webapp/.env
+    depends_on:
+      chat-copilot-webapi:
+        condition: service_started
+  chat-copilot-webapi:
+    image: chat-copilot-webapi
+    build: 
+      context: ..
+      dockerfile: docker/webapi/Dockerfile
+    ports:
+      - 8080:8080
```

Comment:
Chat Copilot WebApi runs on port 40443 by default. Is this changed to 8080 anywhere? Otherwise, how is this accounted for in order for this to work?

Reply:
it is not convient to import dev ssl certificate into another container, so we need to set Kestrel__Endpoints__Http__Url=http://0.0.0.0:8080
 environment variable, the docker-compose file needs have .env file in webapi directory, here is the example for .env content

AIService__Endpoint=
AIService__Key=
Authorization__Type=
Authorization__AzureAd__TenantId=
Authorization__AzureAd__ClientId=
Authorization__AzureAd__Audience=
Authorization__AzureAd__Scopes=access_as_user
Kestrel__Endpoints__Http__Url=http://0.0.0.0:8080
MemoryStore__Type=qdrant

Reply:
Humm...  I see what you are saying. But couldn't we have an ENV command in the webapi dockerfile to set Kestrel__Endpoints__Http__Url=http://0.0.0.0:8080/?

Wouldn't that make it easier?

Reply:
OK, sounds a valid point, I have updated dockerfile to reflect this suggestion, thanks :)

Reply:
Awesome!

Reply:
I had a bit of trouble getting the webapp and webapi talking together so I'll put my .env files here for others to use.

\chat-copilot\webapi\.env
```
AIService__Type=OpenAI
AIService__Key=YOUR_OPEN_AI_KEY
Kestrel__Endpoints__Http__Url=http://0.0.0.0:8080
MemoryStore__Type=qdrant
```

\chat-copilot\webapp\.env
```
REACT_APP_BACKEND_URI=http://localhost:8080/
```

The localhost and 0.0.0.0 were throwing me off.